website.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Lily Kuong Personal Webpage</title>
+    <meta name="author" content="Lily Kuong">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!--Bootstrap-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+    <link rel="stylesheet" type="text/css" href="jquery.fullPage.css" />
+
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+
+
+    <!-- Page Style-->
+    <link rel="stylesheet" type="text/css" href = "styles/style.css">
+    <link rel="stylesheet" type="text/css" media="(max-width:600px" href = "styles/style-small.css">
+    <link rel="stylesheet" type="text/css" media="(min-width:601px" href = "styles/style-large.css">
+
+    <!-- Google font -->
+    <link href="https://fonts.googleapis.com/css?family=Alegreya+Sans+SC|Inconsolata|Merriweather|Open+Sans|Source+Sans+Pro" rel="stylesheet">
+
+    <!-- Font Awesome -->
+    <script src="https://use.fontawesome.com/5544a728a0.js"></script>
+
+
+
+    
+</head>
+<body>
+    <!--Nav Bar -->
+    <nav class="navbar navbar-default navbar-fixed-top">
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand" href="#">LILY KUONG</a>
+            </div>
+            <div class="collapse navbar-collapse" id="myNavbar">
+                <ul class="nav navbar-nav navbar-right">
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#skills">Skills</a></li>
+                    <li><a href="#projects">Projects</a></li>
+                    <li><a href="software_engineer_resume.pdf" target="_blank">Resume</a></li>
```

Comment:
Up to you, but maybe have this at the end so people can click through and get let out at the end